src/constants.ts

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { IconProps } from '@chakra-ui/react';
 import { WindowsLogo, MacosLogo, LinuxPenguin, SourceBranch } from './components/UI/icons';
+import exp from 'constants';
```

Comment:
Do we need this?
```suggestion
```

Reply:
Nope, didnt realize I added that. Nice catch.